HWMgmt/OCPBaselineRedfishService.json

```diff
@@ -0,0 +1,562 @@
+                        "OnlyMemberQuery": {},
+                        "MultipleHTTPRequests": {}
+                    }
+                },
+                "RedfishVersion": {},
+                "RegisteredClients": {
+                    "ReadRequirement": "Recommended"
+                },
+                "ServiceConditions": {
+                    "ReadRequirement": "Recommended"
+                },
+                "ServiceIdentification": {
+                    "ReadRequirement": "Recommended"
+                },
+                "UpdateService": {},
+                "Vendor": {}
+            }
+        },
+        "Session": {
+            "PropertyRequirements": {
+                "ClientOriginIPAddress": {
+                    "ReadRequirement": "Recommended"
+                },
+                "Context": {
+                    "ReadRequirement": "Recommended"
+                },
+                "CreatedTime": {
+                    "ReadRequirement": "Recommended"
+                },
+                "Links": {
+                    "PropertyRequirements": {
+                        "OutboundConnection": {
+                            "ReadRequirement": "IfImplemented"
+                        }
+                    }
+                },
+                "Password": {},
+                "UserName": {}
+            }
+        },
+        "SessionService": {
+            "PropertyRequirements": {
+                "Sessions": {},
+                "SessionTimeout": {
+                    "ReadRequirement": "Recommended",
+                    "WriteRequirement": "Recommended"
+                }
+            }
+        },
+        "SoftwareInventory": {
+            "PropertyRequirements": {
+                "Manufacturer": {},
+                "RelatedItem": {
+                    "ReadRequirement": "Supported"
+                },
+                "ReleaseDate": {
+                    "ReadRequirement": "Recommended"
+                },
+                "ReleaseType": {
+                    "ReadRequirement": "Recommended"
+                },
+                "Updateable": {},
+                "Version": {},
+                "VersionScheme": {
+                    "ReadRequirement": "Recommended"
+                }
+            }
+        },
+        "Task": {
+            "ReadRequirement": "IfImplemented",
+            "PropertyRequirements": {
+                "Messages": {},
+                "TaskMonitor": {},
+                "TaskState": {},
+                "TaskStatus": {}
+            }
+        },
+        "TaskService": {
+            "ReadRequirement": "IfImplemented",
+            "PropertyRequirements": {
+                "DateTime": {},
+                "Tasks": {}
+            }
+        },
+        "UpdateService": {
+            "PropertyRequirements": {
+                "FirmwareInventory": {},
+                "MultipartHttpPushUri": {
+                    "ReadRequirement": "Recommended",
```

Comment:
Any reason to not make it mandatory?

Reply:
I was giving the ability to support "at least one of" the two standard methods.

Reply:
I think we should consider requiring both. If there's a mix of support, then users will have to figure out when to host images on servers versus when to push directly; some users may not have much of a choice in their environment.